xlsx/src/export/worksheets.rs

```diff
@@ -241,6 +242,13 @@ pub(crate) fn get_worksheet_xml(
         ))
     }
     let sheet_data = sheet_data_str.join("");
+
+    for (_ind, merge_cell_ref) in worksheet.merge_cells.iter().enumerate() {
```

Comment:
Can't you just do  `for merge_cell_ref in worksheet.merge_cells {`?

Reply:
Cannot directly use` for merge_cell_ref in worksheet.merge_cells {` as merge_cells is Vec of String,  String gets moved. so doing` for merge_cell_ref in worksheet.merge_cells.clone() {` instead

Reply:
Maybe you can use: `for merge_cell_ref in &worksheet.merge_cells{`? The idea is not to allocate if we don't have to.

Reply:
Oops yes, noted and changed